core/src/main/java/org/apache/calcite/interpreter/Bindables.java

```diff
@@ -269,6 +269,12 @@ public static BindableTableScan create(RelOptCluster cluster,
           .itemIf("projects", projects, !projects.equals(identity()));
     }
 
+    @Override public double estimateRowCount(RelMetadataQuery mq) {
```

Comment:
The push-down fliter is not in the row count calculation, resulting in inaccurate costs